Adding new module

isobar-jm opened the pull request:
Adding module about the operating unit :
For the contact and updating the operation hierarchy logic

isobar-jm committed:
Adding new module

Adding module about the operating unit :
For the contact and updating the operation hierarchy logic

badbole reviewed (changes requested):
just curious... this shuld work .. and is much cleaner:

self.env.user.operating_unit_default_get()

badbole commented:
also.. why is module icrc_helpdesk_mail in this commit?

isobar-jm committed:
Commit

Supressing the module helpdesk from the commit + udating the default operation get from partner
